### PR TITLE
Add sAlert to add/change/remove backlog #1794

### LINKED ISSUE #1794: Notify users about successful add/edit/remove backlog
Now project doesn't have any notice for Backlog manipulation.

Enhancement: Add sAlert after adding, editing and deleting backlog item

Also it makes easy writing test for backlog checking. The operation was successful, if user saw notice with phrase "... backlog item has benn successfully published/edited/eleted "

### PR DESCRIPTION
**Pull request by marla-singer**
Closes #1794

## Changes
- Add sAlert and i18n tags

## Developer checklist
This checklist is to be completed by the PR developer:

- [ ] Alternative solutions were compared/discussed before writing code
    - [ ] trade-offs with this solution are considered acceptable
- [ ] Code in this PR adheres to the [project styleguide](CONTRIBUTING.md)
- [ ] This pull request does not decrease project test coverage
- [ ] If the code changes existing database collection(s), migration has been written
- [ ] If UI texts are added or changed, all texts are internationalized

## Reviewer checklist
Reviewed by: @username1

This list is to be completed by the pull request reviewer:
- [ ] Code works as described/expected
- [ ] Code seems to be error free
    - [ ] no browser console errors visible
    - [ ] no server console errors visible
    - [ ] passes CI build
- [ ] Code is written in a way that promotes maintainability
    - [ ] easy to understand
    - [ ] well organized
    - [ ] follows project coding standards and conventions
    - [ ] well documented

### CODE REVIEW

File: apinf_packages/core/lib/i18n/en.i18n.json [hunk 1]
```diff
@@ -296,6 +298,7 @@
   "deleteBacklogItem_confirmation_text": "Are you sure you want to delete this backlog item?",
   "deleteBacklogItem_cancelButton_text": "Cancel",
   "deleteBacklogItem_deleteButton_text": "Delete backlog item",
+  "deleteBacklogItem_message_backlogRemoved": "The backlog was removed",
```
Comment (matleppa) [inline apinf_packages/core/lib/i18n/en.i18n.json]: "The backlog was removed"
-->
"One backlog item was removed"

File: apinf_packages/core/lib/i18n/en.i18n.json [hunk 2]
```diff
@@ -78,6 +78,8 @@
   "apiBacklogItems_priorityText_High": "High",
   "apiBacklogItems_priorityText_Middle": "Middle",
   "apiBacklogItems_priorityText_None": "None",
+  "apiBacklogItemForm_message_backlogAdded": "The backlog was added",
```
Comment (matleppa) [inline apinf_packages/core/lib/i18n/en.i18n.json]: "The backlog was added"
-->
"One backlog item was added"

File: apinf_packages/core/lib/i18n/en.i18n.json [hunk 3]
```diff
@@ -78,6 +78,8 @@
...
+  "apiBacklogItemForm_message_backlogChanged": "The backlog was changed",
```
Comment (matleppa) [inline apinf_packages/core/lib/i18n/en.i18n.json]: "The backlog was changed"
-->
"One backlog item was changed"

File: apinf_packages/core/lib/i18n/en.i18n.json (hunk 2 above)
Comment (marla-singer) [reply apinf_packages/core/lib/i18n/en.i18n.json]: Why "one"? We don't use "one" anywhere in the same context.
"The" means "This" in the current context. 
Sorry, can't agree with it

Comment (matleppa) [reply apinf_packages/core/lib/i18n/en.i18n.json]: Isn't the idea, that in API card there is a backlog, which contains items.
The items can be added, modified and deleted, not the backlog itself.

Button
image

Modal
image


If you do not like word "one" to be included in alert, we can say e.g. "A backlog item was added"

Comment (marla-singer) [reply apinf_packages/core/lib/i18n/en.i18n.json]: Okay

Comment (marla-singer) [reply apinf_packages/core/lib/i18n/en.i18n.json]: But "The backlog item was changed/removed" because it was deleted the specified item, doesn't it?

Comment (matleppa) [reply apinf_packages/core/lib/i18n/en.i18n.json]: OK.

**Pull request merged**